builtin/game/chat.lua

```diff
@@ -71,9 +80,14 @@ core.register_on_chat_message(function(name, message)
 			core.chat_send_player(name, result)
 		end
 	else
+		local privs = {}
+		for i=1, #missing_privs do
+			privs[missing_privs[i]] = true
+		end
+		local priv_string = format_privs(privs)
```

Comment:
Redundant temporary variable. Consider using `format_privs(privs)` in-line below.